src/Breadcrumbs.php

```diff
@@ -87,21 +86,11 @@ public function withoutEncodeLabels(): self
         return $new;
     }
 
-    /**
-     * Disables rendering of the home item and returns a new instance.
-     *
-     * @return self
-     */
-    public function withoutHomeItem(): self
```

Comment:
Maybe for method `homeItem` use nullable argument for allow:

`$breadcrumbs->homeItem(null)` instead of `$breadcrumbs->homeItem([])` ?

Reply:
Then what about an empty array? It turns out that to disable the home item you can pass an empty array or null, this is not a very good case.

Reply:
Now the property has one type `array` and so it turns out more clearly that if the array is empty, then the home item should not be rendered.

Reply:
We can add validation of array and empty array is invalid value. `null` - empty value. This is more consistently.

Reply:
I'm not sure that it will be clearer to the user this way. Since we can pass an empty array to the `items()` and `options()` methods, but for some reason we can't pass it here. If we leave only the array, it will be consistent with other methods of the class.

Reply:
A single type is preferred.

Reply:
In cases `options()` or `items()` empty array is ok, because this is list of elements and him can be empty.

In case `homeLink()` array is whole definition of link and empty array is invalid link. For empty value in PHP we should use `null`. Not `false`, not empty array, only `null`.

Reply:
OK, you've convinced me.